hopsworks-common/src/main/java/io/hops/hopsworks/common/dao/featurestore/trainingdataset/external/ExternalTrainingDataset.java

```diff
@@ -52,11 +53,7 @@
   private Integer id;
   @JoinColumn(name = "s3_connector_id", referencedColumnName = "id")
   private FeaturestoreS3Connector featurestoreS3Connector;
-  @Basic(optional = false)
```

Comment:
Yes they do, but I moved the name from the `ExternalTrainingDataset.java` to `TrainingDataset.java`. What was happening before was that if it was an external training dataset it was using the column above to store the name, otherwise if it was a HopsFsTrainingDataset it was using the inode name. I created a name column in the TrainingDataset table so that it's easier to access it in the codebase.

Same happened to the Feature groups tables.